src/main/java/edu/ucla/library/prl/harvester/services/HarvestServiceImpl.java

```diff
@@ -125,7 +125,7 @@ public Future<JobResult> run(final Job aJob) {
                         solrResult = Future.succeededFuture();
                     }
 
-                    return solrResult.map(unused -> new JobResult(startTime, records.size()));
+                    return solrResult.map(nil -> new JobResult(startTime, records.size()));
```

Comment:
Curious about this name change. It's not really nothing/nil is it? It's just not being used?

Reply:
Ah, okay, I see what they're saying though that the addition doesn't prevent the other use (so from their design perspective).

Reply:
That's true, it doesn't (how could it though; my proposed change had to be backwards-compatible in order to stay on version 4.x). All it was attempting to do was provide a safer option to use.